### DISCUSSION TITLE
Settings/configuation keep getting reset with updates

### DISCUSSION THREAD

**Discussion by gh0sti** [koush/scrypted#663] [Q&A]
Since I have installed scrypted on my pi in portainer and using watchtower for updates the container keeps getting restored to retaining no configuration/settings. I am not sure what I am doing wrong but my configuration should be kept with each update shouldn't it?

**Comment by koush** [comment]
volume set up incorrectly.

**Comment by gh0sti** [comment]
> volume set up incorrectly.

what do I need to change in the volume to fix then?
